### DISCUSSION TITLE
ant/es/* module not found while building nextjs app

### DISCUSSION THREAD

**Discussion by ziyodbek0300** [ant-design/ant-design#43348] [General]
<img width="631" alt="Screenshot 2023-07-04 at 11 13 18" src="https://github.com/ant-design/ant-design/assets/57743123/c6a1eb96-8f2e-469d-bbca-8cf6dc03f73a">
<img width="592" alt="Screenshot 2023-07-04 at 11 13 08" src="https://github.com/ant-design/ant-design/assets/57743123/0c0ebcae-8d49-448b-b759-6162bf41f801">

**Comment by Wxh16144** [comment]
here: https://github.com/vercel/next.js/pull/52148

**Comment by kiner-tang** [comment]
@ziyodbek0300 You can try to lock the version of nextjs as 13.4.7 until they fix this problem.

**Reply by ziyodbek0300** [reply]
<img width="303" alt="Screenshot 2023-07-04 at 12 14 01" src="https://github.com/ant-design/ant-design/assets/57743123/640636bb-96a4-4fdd-b6c1-fdb7292b9f05">

**Reply by ziyodbek0300** [reply]
Brother We're using 13.4.4 version

**Reply by ziyodbek0300** [reply]
13.4.7 version also cannot solve the problem

**Reply by kiner-tang** [reply]
There is `^` symbol, It will install patch version auto.You can try to set:
```json
"next": "13.4.7"
```
to lock version.

**Reply by kiner-tang** [reply]
You can check the real version in node_modules/next/package.json.

**Reply by ziyodbek0300** [reply]
It's working brother, thanks

**Comment by zombieJ** [comment]
It's `next.js` latest version bug. We can do nothing on antd side.

**Comment by chakri68** [comment]
This worked for me somehow 

```javascript
import ConfigProvider from "antd/es/config-provider";
```

**Reply by tienpd** [reply]
It's work to me. Thank You!
